addon/components/ember-popper-base.js

```diff
@@ -247,10 +247,15 @@ export default class EmberPopperBase extends Component {
 
       this._popper = new Popper(popperTarget, popperElement, options);
 
-      // Execute the onFoundTarget hook last to ensure the Popper is initialized on the target
-      if (isPopperTargetDifferent && this.get('onFoundTarget')) {
-        this.get('onFoundTarget')(popperTarget);
-      }
+      // Execute the registerAPI hook last to ensure the Popper is initialized on the target
+      this.sendAction('registerAPI', {
```

Comment:
I switched to `sendAction` here and didn't trigger any runloops, so not sure why that was happening before. It could be an issue with older Ember, but as we'll be needing this API now for 1.11 we would definitely prefer to do it this way.

Reply:
actually, I think I realize where this would be coming from. Switching to the RAF scheduler should fix this, going to submit that as a separate PR.